codespell_lib/data/dictionary.txt

```diff
@@ -18962,6 +18964,7 @@ laods->loads
 Laotion->Laotian, lotion,
 Laotions->Laotians, lotions,
 laout->layout
+larer->larger
```

Comment:
Please also add `layer`  
`larer->larger, layer,`

Reply:
`Y` is not near `R` on (Qwerty) keyboards. What are the odds of typing `larer` instead of `layer`?

Reply:
Please add both. TIA

Reply:
Done.